jupyter_notebook/static/notebook/js/cell.js

```diff
@@ -73,14 +73,23 @@ define([
             }
         });
 
+        // backward compat.
+        Object.defineProperty(this, 'cm_config', {
+            get: function() {
+                console.warn("warning you are accessing a deprecated field")
```

Comment:
It depends on the browser. But usually you get the file and line-number on the left side of the developper console.